frontend/templates/views/partials/analytics.j2

```diff
@@ -18,6 +18,18 @@
     }
   },
   'triggers': {
+    'default pageview': {
+      'on': 'visible',
+      'request': 'pageview',
+      'vars': {
+        'title': '{{title}}'
+      },
+      'extraUrlParams': {
+        'lcp': '${largestContentfulPaint}',
+        'cls': '${cumulativeLayoutShift}',
+        'fid': '${firstInputDelay}'
+      }
```

Comment:
I would suggest that we add this to a special request rather than `pageview`, since we have discussed that these macros might take a while to resolve, delaying the firing of the request.

Reply:
What kind of special request would be appropriate here?

Reply:
Special, as in individual requests for each CWV:

```
{
  "triggers": {
    "pageviewCls": {
      "on": "visible",
      "request": "endpoint",
      "extraUrlParams": {
        "cls": "${cumulativeLayoutShift}"
      }
    },
    "pageviewLcp": {
      "on": "visible",
      "request": "endpoint",
      "extraUrlParams": {
        "lcp": "${largestContentfulPaint}"
      }
    },
    "pageviewFid": {
      "on": "visible",
      "request": "endpoint",
      "extraUrlParams": {
        "fid": "${firstInputDelay}"
      }
    }
  }
}
```

Reply:
Ah got it - thanks! Do you happen to know what'd be the correct endpoint to use for Google Analytics?

Reply:
I think it would probably be `event`